### ISSUE TITLE
Thank You

### ISSUE THREAD

**Issue by ghost** [mqudsi/syngesture#7]
Seriously thank you so much. Coming from a sway user this is amazing, and the fact its rusty is the proverbial cherry on top of the best cake ever.

**Comment by mqudsi** [comment]
Thank you for taking the time to express your gratitude, @t0m-f00l3ry. It means a lot that other people have found the fruit of this toil beneficial.

To be frank, when I wrote this I was sure I was reinventing the wheel because it seemed like something too simple and too obvious not to exist (a userland daemon implementing touch gestures for the basic driver). It heartens me to see that others also find this useful the same way I did.

I have recently been working on some more under-the-hood improvements to this crate to improve its efficiency (not that it was bad before!) and should have enough to put together a new release soon. It'll bring down CPU usage during touchpad engagement further, and any reduction in CPU usage directly translates to many things relevant to laptop users such as improved battery life, reduced throttling, and a better overall user experience. Keep an eye out!
